### ISSUE TITLE
Thumbnail preview not working

### ISSUE THREAD

**Issue by Saloman1** [zentred/rolimons-deals-finder-to-webhook#2]
hello ur code is a bit outdated and broken see:

> 'thumbnail': {
>               'url': f'https://www.roblox.com/asset-thumbnail/image?assetId={itemid}&width=420&height=420&format=png',
>               }

it has updated to something like: 

> "imageUrl":"https://tr.rbxcdn.com/91ed5b701dfbb4c1212bfc676220e288/30/30/LayeredAccessory/Png"

see here: https://thumbnails.roblox.com/docs#!/Assets/get_v1_assets
im new to coding and not sure how to fix this lol
thank you guy

**Comment by zentred** [comment]
yeah I can't be bothered to update, it's just shitty old code which isn't even useful because people have rolimons snipers to be updated when a deal loads on the page instantly, whereas this only checks every 5 seconds
